worker/src/monitor/pusher.rs

```diff
@@ -0,0 +1,23 @@
+use std::sync::Arc;
+
+use reqwest;
+use tokio::time::sleep;
+
+use crate::{args::WorkerArgs, monitor::collector::MetricsCollector};
+
+pub async fn start_pusher(args: Arc<WorkerArgs>) {
+    let mut metrics_report: MetricsCollector = MetricsCollector::new();
+
+    let client = reqwest::Client::new();
+
+    loop {
+        sleep(args.metrics_report_interval).await;
+        let metrics = metrics_report.get_metrics();
+
+        let _ = client
```

Comment:
This code ignores the error. What if a network error occurs? How should we handle it? We may decide later on.

cc: @gustavodiasag @YuriRDev

Reply:
I'm fine with leaving it as is for now. Filing an issue for a future proper handle might be an option.